script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -126,97 +144,112 @@ I'm ashamed of how forgetful I am.
 -- 「……そうですね。言われてみれば、確かに……」
-"Right... Now that you say it, it does ring a bell..."
+"Oh yeah...
+ You've jogged my memory.
+ In fact, if I think about it..."
 }
 [sha:5ddc808497198e766b29dc97ec80b7794a456156]{
 -- Page 597, Offset 28742.
 -- 子供の時分は危ないからひとりで二階には行かないようにとか、迎賓館には近づくなとか叱られた覚えがある。
-I remember as a child being told not to go to second floor because it was dangerous, or being scolded because I got too close to the guest house.
+I remember being warned never to go to the second floor alone as a child because it was dangerous,
+ and getting scolded for getting too close to the guest house.
 }
 [sha:0fb715000b2ecd5a8dec1237b457b1b8e4090db9]{
 -- Page 597, Offset 28743.
 -- これだけ広い屋敷なんだから、顔を合わせる機会がないまま生き別れた誰かがいたのかもしれない。
-Since the mansion is so vast, there could have been a relative who lived apart and whom I've never had the opportunity to meet.
+In a mansion this large, it's pretty plausible that I could have gone the whole time living here without running into someone.
 }
 [sha:2e9a3ebd92b6ac36d9d179371887bdb918d53d68]{
 -- Page 598, Offset 28744.
 -- 「そうですとも。秋葉さまがお忙しいので今はお暇をだされていますが、乗馬用の馬屋もあったんですから。
-"Quite right! There was also a stable master, though they've been given leave as Akiha-sama is quite busy nowadays.
+"Quite right!
+ Akiha is rather busy these days, so they're currently vacant,
+ but we even have a set of stables for horseback riding.
 }
 [sha:c611478f797e9dc573b1bece3806b72b589ab05f]{
 -- Page 598, Offset 28745.
 -- あの頃のお庭は毎日がお祭りのようで、たいへん楽しゅうございました」
-Back then, the garden was so much fun, every day seemed like a festival!"
+The gardens back then were as bright and colourful as a festival, it was all so much fun!"
 }
 [sha:34f8fccbe4fb5e75201c64fc6f7a7effa882ab93]{
 -- Page 598, Offset 28746.
 -- 昔を懐かしむように笑って、琥珀さんは厨房へ向かった。
-Kohaku-san smiles as though recalling a fond memory before heading to the kitchen.
+Kohaku-san leaves for the kitchen, a nostalgic smile playing across her face.
 }
 [sha:f073dd1bcd26d24c12db40aef383e98c4ca538cc]{
 -- Page 599, Offset 28747.
 -- 「……俺より年上で、何年も前に死んだ養子か……」
-"...An adopted child, older than I and who died years ago..."
+"An adopted older brother who died many years ago..."
 }
 [sha:abbe32dccd28c8b1d42a5ead37a514df7a7f8cd6]{
 -- Page 599, Offset 28748.
 -- ……いや、それこそ琥珀さんの言う通り、考えても仕方のない事だ。
-No, Kohaku-san is right; it's no use thinking about that.
+No, Kohaku-san is right. There's no point thinking any more about it.
 }
 [sha:429f5cc2dd528f4a317696ed244993491cbc9718]{
 -- Page 600, Offset 28749.
 -- ロアの転生先に選ばれ、吸血鬼に変貌して屋敷を去り、街の地下で生き続けた……そんな“誰か”の人生を、俺が<偲|しの>んでも何が戻る訳でもない。
-Being chosen by Roa as vessel, transforming into a vampire, leaving the mansion, surviving under the city... Nothing comes up, as much as I try to bring up memories of %{i}this person's%{/i} life.
+They were chosen as Roa's vessel, transformed into a vampire, left the mansion, and kept on living beneath the city...
+ No matter how much I rack my brains, I can't remember anything about who they could have been.
```

Comment:
I think the second sentence is more about why he shouldn't keep racking his brain about it, or whatever, so it's better that it's like this:
>No matter how much I rack my brains, it won't help me remember anything about who they could have been.